types/node-cron/index.d.ts

```diff
@@ -4,7 +4,11 @@
 //                 burtek <https://github.com/burtek>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export function schedule(cronExpression: string, func: () => void, options?: ScheduleOptions): ScheduledTask;
+export function schedule(
+    cronExpression: string,
+    func: () => void | Promise<void>,
```

Comment:
Technically `any` return value is valid here, I agree with keeping it `void` tho as this enforced clean code :)

Reply:
Doesn't `() => void` cover the `async` case already?

Reply:
According to [Typescript's Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#callback-types) `void` return value should be used for callbacks whose value will be ignored.

Reply:
That's what I meant :)